ReactNativeClient/lib/services/RecordAudio.js

```diff
@@ -0,0 +1,96 @@
+import AudioRecorderPlayer from 'react-native-audio-recorder-player';
```

Comment:
Please check this file and remove the code you're not using. I've noticed dispatch, and the syncCalls and destroy method, which were probably copied and pasted from somewhere else. There might be more so please check.

Also since this is a new file, please use TypeScript as mentioned there: https://github.com/laurent22/joplin/blob/master/CONTRIBUTING.md#coding-style

Reply:
Finally, please move this file to `ReactNativeClient/services` as it's for mobile only (while what's in "lib" is generally shared)